include/verilated.h

```diff
@@ -753,9 +753,9 @@ static inline void* VL_CVT_Q_VP(QData lhs) VL_PURE {
     u.q = lhs;
     return u.fp;
 }
-/// Return QData from void*
-static inline QData VL_CVT_VP_Q(void* fp) VL_PURE {
-    union { void* fp; QData q; } u;
+/// Return QData from const void*
+static inline QData VL_CVT_VP_Q(const void* fp) VL_PURE {
+    union { const void* fp; QData q; } u;
```

Comment:
Yes. The changes are safe to be in master without the rest of commits of this PR.

I sent PR #2650 and #2651 .
Maybe I need to merge master to this PR after these PRs are merged.

Reply:
Feel free to commit when happy & passing.

It can't hurt to merge master into the larger, but if the changes are identical git shouldn't require you to, it will do it when you squash merge the larger patch.

Reply:
You are right! I don't have to merge master to this PR.

Travis looks busy. I will merge #2651 tomorrow.